### ISSUE TITLE
Locale-dependent failures

### ISSUE THREAD

**Issue by eserte** [CpanelInc/Test-MockFile#10]
With a non-English locale (e.g.`LC_ALL=de_DE.UTF-8`) and an older perl (<5.22) the test suite fails:
```
# Failed test '$! text is right.'
# at t/dir.t line 40.
# +--------------------------------+----+--------------------------------+
# | GOT                            | OP | CHECK                          |
# +--------------------------------+----+--------------------------------+
# | Datei oder Verzeichnis nicht g | eq | No such file or directory      |
# | efunden                        |    |                                |
# +--------------------------------+----+--------------------------------+
# Seeded srand with seed '20181025' from local date.
t/dir.t ........... 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/30 subtests 
...
# Failed test '$! text message'
# at t/mock_stat.t line 60.
# Unicode::GCString is not installed, table may not display all unicode characters properly
# +--------------------------------+----+--------------------------------+
# | GOT                            | OP | CHECK                          |
# +--------------------------------+----+--------------------------------+
# | Zu viele symbolische Links (zi | eq | Too many levels of symbolic li |
# | rkulÃ¤r?)                       |    | nks                            |
# +--------------------------------+----+--------------------------------+
# Seeded srand with seed '20181025' from local date.
t/mock_stat.t ..... 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/26 subtests 
```

**Comment by eserte** [comment]
> I'm curious. Why does it succeed above 5.22?

Because localized error messages are now turned on only if `use locale` is in effect.

See perldoc perl5220delta:
>  "$!" text is now in English outside the scope of "use locale"

**Comment by toddr** [comment]
Cool fixed in 8fb9846 and released as 0.006
